R/restore.R

```diff
@@ -584,6 +584,16 @@ installPkg <- function(pkgRecord,
       quiet <- isTRUE(packrat::opts$quiet.package.installation())
       install_local_path(path = pkgSrc, reload = FALSE,
                          dependencies = FALSE, quick = TRUE, quiet = quiet)
+      # if we just installed a binary package, check that it can be loaded
+      # (source packages are checked by default on install)
+      if (identical(pkgType, "binary")) {
+        status <- renv$catch(renv$renv_install_test(pkgRecord$name))
```

Comment:
We've made a change that ensures that the command output isn't automatically printed on error:

https://github.com/rstudio/renv/commit/3cbdd9109bd3eca116f306ae4edb60dff2783450

You might want to use `renv$abort()` instead of `stop()` below to ensure that message gets written to the console still.

Reply:
Took another renv update and used `renv$abort()`.